### PR TITLE
fix(compute): Report compute_backpressure_throttling_seconds as counter

### PR DESCRIPTION
**Pull request by ololobus**
## Problem

It was reported as `gauge`, but it's actually a `counter`.

Also add `_total` suffix as that's the convention for counters.

The corresponding flux-fleet PR: https://github.com/neondatabase/flux-fleet/pull/386

### CODE REVIEW

File: compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   metric_name: 'compute_backpressure_throttling_seconds',
-  type: 'gauge',
+  type: 'counter',
```
Comment (ololobus) [inline compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet:3]: @knizhnik am I right that `neon.backpressure_throttling_time()` is actually a counter internally? I.e. it never goes down

**Review by tristan957 — COMMENTED** [review]
Thanks for catching this. Maybe we should rename it like @mickael-carl said as well so it is more clear.

**Review by knizhnik — APPROVED** [review]

File: compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   metric_name: 'compute_backpressure_throttling_seconds',
```
Comment (mickael-carl) [inline compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet:2]:
```suggestion
  metric_name: 'compute_backpressure_throttling_seconds_total',
```

**Review by mickael-carl — APPROVED** [review]
Counters need to have a `_total` suffix

File: compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet (hunk 2 above)
Comment (ololobus) [reply compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet]: Renamed the file name as well for consistency + flux PR https://github.com/neondatabase/flux-fleet/pull/386

**Review by tristan957 — APPROVED** [review]

**Pull request merged**